apparmor.d/groups/akonadi/akonadi_akonotes_resource

```diff
@@ -38,10 +38,12 @@ profile akonadi_akonotes_resource @{exec_path} {
   owner @{user_config_dirs}/kdedefaults/kwinrc r,
   owner @{user_config_dirs}/kdeglobals r,
   owner @{user_config_dirs}/kwinrc r,
+  
+  @{sys}/devices/pci[0-9]*/**/{device,subsystem_device,subsystem_vendor,uevent,vendor} r,
```

Comment:
This is already present in `abstractions/dri-enumerate`

Reply:
Oops. I had collected these rules over several weeks and missed that you added dri-enumerate in the meantime.